src/sagemaker/pytorch/estimator.py

```diff
@@ -210,9 +212,7 @@ def __init__(
                     To learn more, see `Training with parameter servers
                     <https://sagemaker.readthedocs.io/en/stable/frameworks/tensorflow/using_tf.html#training-with-parameter-servers>`_.
 
-                **To enable distributed training with
```

Comment:
do we need to make changes for compiler here?

Reply:
No, but it didn't look good, just wanted to fix it quick. Shall we fix it? :)

Reply:
lets keep it since its documentation changes, generally for code changes we don't change unrelated parts in a single PR, because in the future people will found compiler code has been touched by this PR(but its only for torchrun.)

Reply:
but thanks for that!